components/inputs/docs/input-text.md

```diff
@@ -27,6 +27,7 @@ The `<d2l-input-text>` element is a simple wrapper around the native `<input typ
 | `disabled` | Boolean | Disables the input |
 | `input-width` | String, default: `100%` | Restricts the maximum width of the input box without impacting the width of the label |
 | `label-hidden` | Boolean | Hides the label visually (moves it to the input's `aria-label` attribute) |
+| `label-secondary` | String | Secondary offscreen label for accessibility |
```

Comment:
I guess `description` doesn't quite work the way you want it? It also creates an offscreen thing that hooks it up using `aria-describedby`.

Reply:
Yeah, I'd consider description briefly, but it's also leveraged by the tooltip when invalid.  It could potentially be used, but not perfectly since one would have to take precedence.

Reply:
I've done a bit more experimenting with this using VO and NVDA.  We could actually use the `description` and similarly insert the user-specified value into the validation message for the invalid case.  

Reasons to use `label-secondary`:
1. `aria-describedby` value is not displayed in the list of form controls, for example the VO Rotor.  In this case, having this part of the label is helpful, but not essential. 
2. the value doesn't get announced until after the screen reader announces a bunch of other stuff... which works, but not ideal.

Reasons to use `description`:
1. it doesn't add a seemingly redundant property to the component API
2. consumers might mis-use `label-secondary`, by putting verbose descriptions that should be put into the aria description 

I think I prefer the UX of `label-secondary`, but I am more worried about consumers using it incorrectly, so I _think_ I am leaning towards changing this to just use `description`.  I'd also considered always rendering the user-specified description as I have done here (offscreen part of label), but I don't think we want potentially verbose descriptions showing up as part of labels in the list of form fields such as in VO Rotor.

Reply:
Cool yeah seems like you've considered the trade-offs -- I'm good with either approach. I do think this could become an interesting pattern that we could use in many places... inputs that have their units displayed right in the input like this. Other examples that come to mind: entering CSS values like `px` or `em` (in a branding UI), currency input, HEX colour values...

Which is making me think -- could the `%` thing just be in a different slot that we introduce? The slot could end up inside the label in the DOM, but we just position it to the right/left similar to the existing slot you've used. That way you may not need to introduce a new lang term for "percent" it would just be "label value %". 🤷